viya_ark_library/k8s/test_impl/response_data/resources_secrets.json

```diff
@@ -0,0 +1,102 @@
+                "tls.key": "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcGdJQkFBS0NBUUVBeHlKKzJLYThwdm1uUlFtU0pwRmxBYTh3QXNpZ1Y2UHZFSnZ1MjF4TS9UUnhyWkVxClRWM3QrOWF3RjkvVUVKNk5JNlFkRlBhcXBoS3p5UEdtc01RcXNzVGFBT3JlY2dkdkcyTTd4Wk1ldzZXVXFpNVAKbm9CVU91S3ZiMW5heXJaMWU1d1pybjVNaVppSU9XWkdZVXJHbEh1OWMvb1NUMnpHbHo1TmttN2pMUWtZanlFagppS0hkaGFvRjB6bU9jZ1FYRkpDQWhwVUpkZkwzTTQxVGhiVXZjVmoyNk5EUXA3SFFUSlVPM05PWkJhRU4vMUNMCnNBalpJdmZvS0NGVmlHVGJ0KzZwVE4rRXgrbXgvUi9rVGhmS1kwSTFkVEROQlRPNDdOR1ZVMjhwWkZRZS9hTHcKdWI4Z2xycFVVbjVhNnB1RExXSWxGUFNwNmU1VTU5UTZsR2F4Q3dJREFRQUJBb0lCQVFDSU5FWUN0Zmt2Vkc5ZQp2eThpQml6S1BDTmdwQTFZSko3dUExb2EzRVVrbzJ4bHhYYmp1QzJCSmRVSVgyWlRmNzhWODdpWWtDY2t0SUlZCkhZRmdRcXpaM2Q4bTVYOEZDNlNwWFBTVExRL0x6a09qSzAzMklkeDY4bllBZWJzM0VCbkhWSHdVWi9xOTJvREMKa2Z2dHJCR3JtZjk1aGtCajV3am1oZ3pVN2RlaEF2U25oSEp1UzkvU3lsaytCVzN1RFlIUFFzWU9MZ1B5MXJlUwpZcVZhT2krQjlkZFNUakFmNWtNeXBjdTlQcURWSUsyTSszdGNUbklvV3cyK0NhMDd3SU9LRUdyL3JqdHpTbnFWCnNnYnBtaXVnZmJOZTdjYTNFbjl5b3ZtdXBPSXlhNld4R3UybDBiRkhpdjVFaUFVNFg3ZUQ4aFJaU1o1bXp1RTgKTnZxVEN1Q2hBb0dCQU5wQUQ5Vkg1OFd3UzQ5T3RJemtIa0F1SncwbUhjbW1YRkdRK3Rqdy9aVFdlSE1ubkNaYQovKzBGNUhEWkhrYjdGNUQvd1ZBK3FadVFpeHZyTHpScS9RV0VKOE9SYW5rbkVFL1FxSFlvbjU0QU5WVisycllaCmdubUhCWmFPVnNiTjRqeFh3ZEN1YVQrY1lndFgxOFZENXlyMHNhNFBWREwxSXo5OUFsS2hENkp6QW9HQkFPbVUKQS96c293Z1JRTldScHFUYlZKTnk5NDRUYktZaW5Gc0xJSjF2aXBwZFFtUzFnekJLTnM5aXBTaW8xVTB0UXpkegp0dnFGa1MrZkJ5UlhPRGZ6V2ZsN09MQXByQU5GS21acEJhejRHMUZBejZkdWRFMTNqdjJDYUFvNmNPbHFrQW5qCnMreWJaSXk3NlRVd0JoaWxycnl3MXN1WCtmSXBpejhGYWlyZklGa0pBb0dCQUtNNXdoMnExMlo4UVNaK29IMjIKcGdFdUF6ck1SUDdwRXFaV0hhVmVKdWpIdGZHNXNBam5pZnhFcWVMVUh0YkxONGJVSUVuWmM1UEsrSjc2dDZVbQpWdkZuTXk0S2RXMENrQ3FCVFZ5QmFVOExyTmV5TmZSM0E2UE1IWnZKL0pCV3JscWU3MjhZNGRJWExTUjZuLzhDCmw4TUNtbGdwVXJtQnFNajB2Mlg0MnU2TkFvR0JBSWI4UlRtN3Y1UC85SW4zL0dnMVlMRTl0WmJqM3R3OEVFUUYKTURvTHBmd0RmVVVhS05DVmN3dHlwS1BVdmtBMXNpeXZzTVVpdjM3WDFwYkgyUE5zclVpaVQyWnZEcnR0RTAzUAp6N25nUUZrNnJVYkRiRzhBY3lCS1pyblNnRUpFbFMxQjdFOEdscHRHVlVXcGxUeFZNNkJVU2kycWpJWElRSEtSClBJTytJc0dSQW9HQkFJRGU3a28vUzdEYUpCa1BFN0dtMVQ1SmF0MGdwUGVQc3hUMjI5bjVZanRQZ3Bud1V1QlEKKytCajBMRkNNWEJjcDlqTCtvcFF3WVk1VWlhUmlyY2JDTnVYd0NTOGtCMjFNcHlXdXhmdGtGZXllY0pIS1ZVcQo4SnVIRU83TTFDRiswSmdpQmJYbERGU1U3ZHE2Tnp3MENQYTJ2M3J2d20rZVI5Ymo0cWx2d1RYdgotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo="
+            },
+            "kind": "Secret",
+            "metadata": {
+                "annotations": {
+                    "cert-manager.io/alt-names": "localhost,sas-web-data-access,sas-web-data-access-78b5f77db6-nr8zv",
+                    "cert-manager.io/certificate-name": "sas-web-data-access-78b5f77db6-nr8zv",
+                    "cert-manager.io/common-name": "sas-web-data-access-78b5f77db6-nr8zv",
+                    "cert-manager.io/ip-sans": "10.104.211.55,10.254.1.128,127.0.0.1,192.168.254.179",
```

Comment:
These are probably dynamic IPs, but since some don't seem to be local, it might be best to clear out this field or put in something innocuous like 127.0.0.1

Reply:
updated.